src/gc/impl/conservative/gc.d

```diff
@@ -2871,8 +2871,12 @@ struct Gcx
             if (scanThreadData[idx].tid != scanThreadData[idx].tid.init)
                 startedThreads++;
 
+        version (Windows)
+            alias allThreadsDead = thread_DLLProcessDetaching;
+        else
+            enum allThreadsDead = false;
```

Comment:
So what happened here? A DLL with running threads was unloaded, terminating them (causing them to segfault)?

Reply:
No, the process terminates while threads are still running. The threads are just terminated as well without `DllMain(DLL_THREAD_DETACH)` being called. `DllMain(DLL_PROCESS_DETACH) is still ìnvoked, though.